### PR TITLE
Reload Cart Issue Resolved

### PR DESCRIPTION
**Pull request by Tejas2909**

### CODE REVIEW

File: client/src/components/Cart/Total.js [hunk 1]
```diff
@@ -68,8 +71,12 @@ const Total = (props) => {
             total: props.price * 100,
             user: state.user
           });
+
           console.log(captureResponse.data);
```
Comment (pal-16) [inline client/src/components/Cart/Total.js:75]: Please remove the log

**Review by pal-16 — CHANGES_REQUESTED** [review]
Just go through the files for trailing logs

File: client/src/components/MyOrders/MyOrders.js [hunk 2]
```diff
@@ -33,46 +35,35 @@ const MyOrders = () => {
         }
       }
     );
+    // console.log(fetchedOrders.data.data);
```
Comment (ShubhankarKG) [inline client/src/components/MyOrders/MyOrders.js:38]: Also this one log

File: client/src/components/MyOrders/TestBook.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from "react";
+import { Paper } from "@material-ui/core";
+import OrderedItem from "./OrderedItem";
+const TestBook = (props) => {
+  useEffect(() => {
+    console.log(props.checkTest.products);
```
Comment (ShubhankarKG) [inline client/src/components/MyOrders/TestBook.jsx:6]: Please remove this log as well

File: client/src/components/MyOrders/TestBook.jsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            backgroundColor: "rgb(241, 241, 241)",
+            width: "60%",
+            display: "block",
+            margin: "100px 0!important"
+          }}
+        >
+          <OrderedItem
+            title={`${product.test.subject} Full Course`}
+            price={product.price}
+            //   buyDate={product.course.applicableExamDate}
```
Comment (ShubhankarKG) [inline client/src/components/MyOrders/TestBook.jsx:42]: Please remove the comment if not needed

**Review by ShubhankarKG — CHANGES_REQUESTED** [review]

File: client/src/components/MyOrders/TestBook.jsx (hunk 4 above)
Comment (Tejas2909) [reply client/src/components/MyOrders/TestBook.jsx:42]: Actually this is for date field for the test series which is not yet added . Once added I would uncomment It

Comment (Tejas2909) [reply client/src/components/MyOrders/TestBook.jsx:39]: Done!!

**Comment by Tejas2909** [comment]
Done with the changes

**Review by ShubhankarKG — APPROVED** [review]

**Pull request merged**